docs/components/datasource/finngen/_finngen.md

```diff
@@ -0,0 +1 @@
+# FinnGen
```

Comment:
I wouldn't be surprised if there is a more systematic way, but you always need to face the problem of "prettifying" the output. For example writing `FinnGen` instead of the Python conventions.